internal/build/version.go

```diff
@@ -0,0 +1,6 @@
+// Package build handles information and processes about the k6 binary's build.
+package build
+
+// Version contains the current version of k6
+// represented using Semantic Versioning expression.
+const Version = "0.56.0"
```

Comment:
I believe it's not up-to-date
```suggestion
const Version = "0.57.0"
```